src/sections/footer.html

```diff
@@ -1,4 +1,48 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="lg:hidden text-darkgreen text-base md:text-base font-medium">
```

Comment:
The footer text is not meant to be green.
```suggestion
<div class="lg:hidden text-base md:text-base font-medium">
```